lldb/test/API/lang/swift/expression/self_from_register/TestSelfFromRegister.py

```diff
@@ -0,0 +1,17 @@
+# TestSelfFromRegister.py
```

Comment:
I know that this is about self in a register, but the way the test is written we really have no control over that. So lets call it TestSwiftProtocolExtensionSelf or something like that.